CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributor's Guide
+
+HOW TO CONTRIBUTE TO OPEN SOURCE accepts PR's(pull requests) from **newbies** only, this is to help **newbies** get familiar with contribution processes.
+
+Issues can be submitted by anyone, seasoned developers or newbies.
+
+# Getting Started
+
+1.  Fork the project on Github.
+2.  Clone the project.
+3.  Create a branch specific to the issue you are working on.
+    
+    ```
+    git checkout -b update-readme-file
+    ```
+    For clarity to yourself and others on the issue you're working on, name your branch something like `update-xxx` or `fix-xxx` where `xxx` is a short description of the changes you're making. For example `update-readme` or `fix-typo-on-contribution-md`.    
+4.  Make your changes in the new git branch then add your modified files to git: 
+    
+    ```
+    git add path/to/filename.ext
+    ```
+
+    You can also add all unstaged files using: 
+    
+    ```
+    git add .
+    ``` 
+
+    Note, using a `git add .` will automatically add all files. You can do a `git status` to see your changes, but do it before `git add`.
+
+5.  Commit your changes using a descriptive commit message.
+    
+    ```
+    git commit -m "Brief Description of Commit"
+    ```
+6.  If you made more than one commit, you should squash your commits.
```

Comment:
Should this be required for new contributors? I think squashing a commit falls outside the basic git usage territory and we should not impose this at the outset. @shubheksha Thoughts?

In case this is kept, a description of how to do it should help.